### DISCUSSION TITLE
Release v1.14.4 / 5.69.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4079] [Announcements]
This releases fixes a few issues with 1.14.3 and provides a production ready 1.14.x build.

For a full list of changes and fixes please review the [full change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1144--5694---2024-07-13).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.4'>Release v1.14.4 / 5.69.4</a>.</em>

**Comment by rsherry1966** [comment]
I paid for this software on 10/20/09 and it was MARVELOUS THEN and I continue to use it today, but am trying this new version.  Anxious to try it now, BUT, it keeps asking me for some code on installation.  My Serial Number from 2009 does not work in this.  Why?  Thanks.

**Reply by bastik-1001** [reply]
So far the installer never asked me for any code, please make sure you use the official binaries provided by the project.

This project is not an official continuation of the original product. If you take a look at the [project history](https://github.com/sandboxie-plus/Sandboxie#-project-history), you will notice that the original developer handed over to Invincea, and they handed it over to Sophos. The conditions are unknown, but don''t apply here, anyway. Sophos released the source code, seemingly no longer developing the software, @DavidXanatos forked the source code to provide Sandboxie Classic (older user interface) and Sandboxie-Plus (new user interface).

All the features that required to pay for the software, as those sources were released, are freely available in any version released. David added new features, he wants users to obtain a license/certificate for. In most instances, they are purchased by the user. It's possible to get a certificate by [contributing to the project](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md#non-monetary-contributions), so without having to pay for it.

**Comment by PicchiarelloMatto** [comment]
Why is this version not yet published on the official website?
Is it better to stay with version 1.13.7 until it is published on the official site?

**Reply by bastik-1001** [reply]
The website and the updater are intentionally delayed, mostly to avoid shipping versions with errors that have not been triggered by tests, to a larger set of users. Sometimes users, that install the versions from GitHub, report errors.

I assume that the bug(s) "Administrators cannot change the sandbox configuration" and maybe "Start Restrictions tab's layout is broke" that are present in 1.14.3, but not in 1.13.7, made David not promote it on the website. 1.14.4 should be the next version that gets listed there, but either not enough time has passed or David did not have the time to update the website.

So far, there have been no reports of something being wrong with 1.14.4; there should be no reason to stick to 1.13.7. There is a bug that is not fixed in 1.14.4, which is not present in 1.13.7, namely #4074, but it only triggers if the non-default list-view is used.

**Reply by PicchiarelloMatto** [reply]
Ok thanks, the doubt came to me because version 1.13.7 continues to report the new version available, but then refers to the official site where it's not present, while with previous versions the installation occurred automatically once confirmed without the need for having to download it manually from the site.

**Reply by bastik-1001** [reply]
You are welcome, then it seems that David did not have the time to update the website, as this is something he should do, if he lets the updater announce the new version.

Some versions ago, there was a change in the updater, to have it report stable versions, but it not being installed automatically, unless the user has a valid active certificate. It is less convenient for most users, but gives another reason to consider getting a certificate. The updater still installs pre-releases for everyone, since those might contain more bugs and when a user opts for it, he can help out by testing and gets some features before they are released in the stable version.

**Reply by DavidXanatos** [reply]
1.14.4 is now on the website, i consider it quite stable

next will be 1.14.5 and should be out very soon, also as atable

**Comment by BilleBarrett** [comment]
Can anyone please help me with my issue with Adobe, everytime I click on a PDF file and it open it is not in the Sandbox
I have tried several things and I still can't get PDF's into the sanbox
Thank you
Bill e. Barrett
